Dockerfile.ci.s390x

```diff
@@ -0,0 +1,297 @@
+RUN curl -sSLf \
+    "https://github.com/twitter/twemproxy/releases/download/0.5.0/twemproxy-${TWEMPROXY_VERSION}.tar.gz" \
+    | tar -C "/home/${USER_NAME}" -xz
+
+ARG TWEMPROXY_CONFIGURE_OPTIONS
+ARG TWEMPROXY_BUILD_OPTIONS
+ARG TWEMPROXY_PREFIX=/opt/twemproxy
+RUN cd ~/twemproxy-"${TWEMPROXY_VERSION}" \
+    && autoreconf -fvi \
+    && ./configure --prefix="${TWEMPROXY_PREFIX}" ${TWEMPROXY_CONFIGURE_OPTIONS} \
+    && make ${TWEMPROXY_BUILD_OPTIONS}
+RUN cd ~/twemproxy-"${TWEMPROXY_VERSION}" \
+    && sudo make install \
+    && rm -rf ~/twemproxy-"${TWEMPROXY_VERSION}"
+
+# Redis installation
+ARG REDIS_VERSION=5.0.3
+RUN curl -sSLf "https://github.com/antirez/redis/archive/${REDIS_VERSION}.tar.gz" \
+    | tar -C "/home/${USER_NAME}" -xz
+
+ARG REDIS_BUILD_OPTIONS
+RUN cd ~/redis-"${REDIS_VERSION}" && make ${REDIS_BUILD_OPTIONS}
+ARG REDIS_PREFIX=/usr/local
+RUN cd ~/redis-"${REDIS_VERSION}" \
+    && sudo make PREFIX="${REDIS_PREFIX}" install \
+    && rm -rf ~/redis-"${REDIS_VERSION}"
+
+# not really meant to be changed
+ARG RBENV_ROOT="/home/${USER_NAME}/.rbenv"
+ENV RBENV_ROOT="${RBENV_ROOT}"
+ARG RBENV_BINPATH="${RBENV_ROOT}/bin"
+ARG RBENV_PATH="${RBENV_ROOT}/shims:${RBENV_BINPATH}"
+ARG RBENV_RUBYBUILD_ROOT="${RBENV_ROOT}/plugins/ruby-build"
+
+# update to git clone to include --shallow-submodules --no-tags --jobs $(nproc)
+# when newer git versions land in the base distro
+RUN git clone --recurse-submodules --depth 1 --no-checkout --progress \
+    https://github.com/rbenv/rbenv.git "${RBENV_ROOT}"
+
+RUN git clone --recurse-submodules --depth 1 --no-checkout --progress \
+    https://github.com/rbenv/ruby-build.git "${RBENV_RUBYBUILD_ROOT}"
+
+ARG RBENV_TAG=v1.1.2
+# update git fetch to include --jobs $(nproc) when newer git versions land
+RUN cd "${RBENV_ROOT}" \
+    && (git checkout -q -f "${RBENV_TAG}" 2> /dev/null \
+    || (git fetch --prune --depth 1 origin tag "${RBENV_TAG}" \
+    && git checkout -q -f "${RBENV_TAG}"))
+
+ARG RBENV_RUBYBUILD_TAG=v20210825
+RUN cd "${RBENV_RUBYBUILD_ROOT}" \
+    && (git checkout -q -f "${RBENV_RUBYBUILD_TAG}" 2> /dev/null \
+    || (git fetch --prune --depth 1 origin tag "${RBENV_RUBYBUILD_TAG}" \
+    && git checkout -q -f "${RBENV_RUBYBUILD_TAG}"))
+
+RUN sudo apt update -y
+RUN cd "${RBENV_ROOT}" && src/configure && make -C src || true
+
+RUN echo -n "export PATH=${RBENV_BINPATH}" >> ~/.bash_rbenv \
+    && echo ":${HOME}/.local/bin:$PATH" >> ~/.bash_rbenv \
+    && echo 'eval "$(rbenv init -)"' >> ~/.bash_rbenv \
+    && echo 'source ~/.bash_rbenv' >> ~/.profile
+
+ENV PATH="${RBENV_PATH}:/home/${USER_NAME}/.local/bin:${PATH}"
+
+ARG MRI_DEPS="bzip2 zlib1g-dev libreadline-dev libssl-dev libffi-dev libgdbm-dev libncurses-dev"
+RUN test "x${MRI_DEPS}" = "x" || sudo apt install -y ${MRI_DEPS}
+
+RUN echo 'gem: --no-document' >> ~/.gemrc \
+    && echo --color > ~/.rspec
+
+ARG CONFIGURE_OPTS=--disable-install-doc
+RUN mkdir -p ~/.local/bin \
+    && echo  "#!/bin/bash\n\n" \
```

Comment:
These `echo` commands originally used the `-e` switch - is it not working correctly for you?

I just ran (on x86_64) the following, which shows `-e` to work well and removing it to quote the `\n`s rather than interpreting them:
```shell
apisonator$ podman run --rm -ti ubuntu /bin/bash -c 'echo -e "hello\n"; echo "hello\n"'
hello

hello\n
apisonator$
```